docs/demos/sequential_testing.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>This should almost certainly be significant because of our simulation setup. I've run this simulation and I get the p-val at the lower bound of our range because the context dependence is so strong/pure. It would be nice to get similar results here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>